emc/dmri.py

```diff
@@ -0,0 +1,96 @@
+"""Representing data in hard-disk and memory."""
+import attr
+import numpy as np
+import h5py
+from pathlib import Path
+
+
+@attr.s(slots=True)
+class DWI:
+    """Data representation structure for dMRI data."""
+
+    dataobj = attr.ib(default=None)
+    """A numpy ndarray object for the data array, without *b=0* volumes."""
```

Comment:
Is it just me, or is it weird that these strings appear _after_ the attr they refer to?

Reply:
It's a rejected PEP which sphinx honors (so these attributes will indeed appear documented)